pkgs/games/tuxtyping/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv , fetchurl, pkgconfig, SDL, SDL_image, SDL_mixer, SDL_ttf, librsvg }:
+
+stdenv.mkDerivation rec {
+  version = "1.8.0";
+  name = "tux-type-${version}";
+
+  src = fetchurl {
+    url = "https://sourceforge.net/projects/tuxtype/files/tuxtype-source/TuxType%20${version}%20-%20Source/tuxtype_w_fonts-${version}.tar.gz/download";
```

Comment:
This seems to be the upstream now https://salsa.debian.org/tux4kids-pkg-team/tuxtype and they have 1.8.3:

https://salsa.debian.org/tux4kids-pkg-team/tuxtype/blob/debian/doc/ChangeLog

Reply:
So I tried switching to 1.8.3 and it added a dependency on SDL_net (what??) and then tried to access a git repository that doesn't exist anymore...

Reply:
Can you share the expression or the error?

Reply:
Sure:

checking for libxml-2.0... yes
checking for t4k_common... no
configure: t4k_common not located by pkg-config, checking with AC CHECK LIB:
checking for InitT4KCommon in -lt4k_common... no
configure: error: libt4k_common is required: git://git.debian.org/tux4kids/t4kcommon.git

Reply:
I see the repository has moved to https://salsa.debian.org/tux4kids-pkg-team/t4kcommon but doesn't really seem to have anything of value in it.

I'm also very confused about the SDL_net dependency.

Reply:
I found it here https://github.com/tux4kids/t4kcommon

Reply:
Looks like debian did some weird stuff in some of their branches (several 1.8.3-x releases just to change notes). I grabbed the code from a branch marked as upstream, so we'll go with that.

Reply:
1.8.3 is the upstream release, dash releases are just packaging changes.